spec/connect/client_spec.rb

```diff
@@ -292,9 +306,17 @@
       subject.register!
     end
 
-    it 'should call activate_product on api' do
-      allow(System).to receive_messages(credentials?: true)
-      expect(subject).to receive(:activate_product)
```

Comment:
Why did this spec go? I think it's useful to check if `activate_product` was called on the API.

Reply:
Ah, nevermind -- I see it has moved to the specs down below.